### PR TITLE
small pull request of formatting, consistency and lowercase :)

### PR DESCRIPTION
**Pull request by xyzze**
made changes to the atlas-config.bat

NIC settings was not changed, reverted this part after

### CODE REVIEW

**Review by uncognic — APPROVED** [review]
good

File: src/AtlasModules/atlas-config.bat [hunk 1]
```diff
@@ -427,11 +427,8 @@ powercfg /s 11111111-1111-1111-1111-111111111111
 if %ERRORLEVEL%==0 (echo %date% - %time% PowerPlan Imported...>> %WinDir%\AtlasModules\logs\install.log
 ) ELSE (echo %date% - %time% Failed to import PowerPlan! >> %WinDir%\AtlasModules\logs\install.log)
 
-:: Set SvcSplitThreshold
-:: Credits: revision
-for /f "tokens=2 delims==" %%i in ('wmic os get TotalVisibleMemorySize /format:value') do set mem=%%i
-set /a ram=%mem% + 1024000
-reg add "HKLM\System\CurrentControlSet\Control" /v "SvcHostSplitThresholdInKB" /t REG_DWORD /d "%ram%" /f
+:: set SvcSplitThreshold
+reg add "HKLM\System\CurrentControlSet\Control" /v "SvcHostSplitThresholdInKB" /t REG_DWORD /d "ffffffff" /f
```
Comment (he3als) [inline src/AtlasModules/atlas-config.bat:431]: Have you tested this?

Comment (xyzze) [reply src/AtlasModules/atlas-config.bat:431]: yes

**Review by he3als — APPROVED** [review]

File: src/AtlasModules/atlas-config.bat (hunk 1 above)
Comment (he3als) [reply src/AtlasModules/atlas-config.bat:431]: okay nice

Comment (xyzze) [reply src/AtlasModules/atlas-config.bat:431]: it is useless though

Comment (xyzze) [reply src/AtlasModules/atlas-config.bat:431]: it is useless though

**Pull request merged**